cli/lib/command/internal_command_settings.dart

```diff
@@ -4,17 +4,67 @@ import 'package:stax/context/context.dart';
 import 'package:stax/settings/settings.dart';
 
 class InternalCommandSettings extends InternalCommand {
+  static const availableSettings = {
+    'branch_prefix': 'Prefix to add to all new branch names (e.g., "feature/")',
```

Comment:
Let's put description field into `cli/lib/settings/setting.dart` class, and here we will just have a list of available commands. 
```
static const availableSettings = [
  Settings.instance.branchPrefix.
]
```